src/order/game_add.lean

```diff
@@ -33,14 +33,17 @@ variables {α β : Type*} (rα : α → α → Prop) (rβ : β → β → Prop)
 
 namespace prod
 
-/-- The "addition of games" relation in combinatorial game theory, on the product type: if
-  `rα a' a` means that `a ⟶ a'` is a valid move in game `α`, and `rβ b' b` means that `b ⟶ b'`
-  is a valid move in game `β`, then `game_add rα rβ` specifies the valid moves in the juxtaposition
-  of `α` and `β`: the player is free to choose one of the games and make a move in it,
-  while leaving the other game unchanged. -/
+/-- `prod.game_add rα rβ x y` means that `x` can be reached from `y` by decreasing either entry with
+  respect to the relations `rα` and `rβ`.
+
+  It is so called, as it models game addition within combinatorial game theory. If `rα a₁ a₂` means
+  that `a₁ ⟶ a₂` is a valid move in game `α`, and `rβ b₁ b₂` means that `b₁ ⟶ b₂` is a valid move
```

Comment:
This maybe a bit counterintuitive, because in mathlib we use `<` rather than `>`. In terms of games, `a₁ < a₂` means `a₁` is simpler than `a₂`, and [pgame.subsequent](https://leanprover-community.github.io/mathlib_docs/set_theory/game/pgame.html#pgame.subsequent) and [pgame.is_option](https://leanprover-community.github.io/mathlib_docs/set_theory/game/pgame.html#pgame.is_option) have argument orders consistent with this. So it would be better to keep the original order, saying `rα a₁ a₂` means that `a₂ ⟶ a₁` is a valid move. (You might restore the original notation `a` and `a'` if you want.)

Reply:
I didn't change the order, this was just a docstring typo. Also, will add the placeholder PR shortly.

Reply:
Sorry, what was the typo? And why do you say you didn't change it? `pgame.is_option a' a` means there's a valid move from `a` to `a'`, not the other way around, right? So after your modification, the docstring makes wrong sense when applied to `rα := pgame.is_option`.

Reply:
I didn't redefine the function, just renamed variables in an attempt to match `is_option` and such, then accidentally swapped variables in the docstring.

Reply:
Yeah, my comment is on the docstring from the start. Looks good now, thanks!

maintainer merge